plugins/woocommerce/src/Admin/Features/Settings.php

```diff
@@ -42,11 +37,6 @@ public function __construct() {
 		}
 
 		add_filter( 'woocommerce_settings_features', array( $this, 'add_feature_toggle' ) );
```

Comment:
We've removed the toggle option, should we remove this too?

Reply:
Good call, removed in https://github.com/woocommerce/woocommerce/pull/50977/commits/32d58de6f613defe82ee754222324410bdbe3774